pyface/workbench/debug/debug_view.py

```diff
@@ -30,9 +30,7 @@ class DebugViewModel(HasTraits):
     # 'Model' interface.
     # ------------------------------------------------------------------------
 
-    @on_trait_change(
-        "window.active_editor", "window.active_part", "window.active_view"
-    )
+    @observe("window.active_editor,window.active_part,window.active_view")
     def refresh(self):
```

Comment:
This is also a bug and not just a style recommendation

```suggestion
    def refresh(self, event):
```